From 8e0c60a7eaa8b06c14ce04c6d384809715894526 Mon Sep 17 00:00:00 2001
From: Hideyuki Yamashita <yamashita.hideyuki@po.ntt-tx.co.jp>
Date: Mon, 5 Nov 2018 22:14:14 +0900
Subject: [PATCH 09/10] docs: add label for usecase to be referred

Add a label to `Shutdown SPP VF Components` section to be referred.

Signed-off-by: Hideyuki Yamashita <yamashita.hideyuki@po.ntt-tx.co.jp>
Signed-off-by: Naoki Takada <takada.naoki@lab.ntt.co.jp>
---
 docs/guides/spp_vf/use_cases/usecase1.rst | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/docs/guides/spp_vf/use_cases/usecase1.rst b/docs/guides/spp_vf/use_cases/usecase1.rst
index 711df03..4a8f4bc 100644
--- a/docs/guides/spp_vf/use_cases/usecase1.rst
+++ b/docs/guides/spp_vf/use_cases/usecase1.rst
@@ -282,6 +282,8 @@ Now, you can login to VMs from the remote host1.
     # spp-vm2 via NIC1
     $ ssh sppuser@192.168.150.32
 
+.. _spp_vf_use_cases_usecase1_shutdown_spp_vf_components:
+
 Shutdown SPP VF Components
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- 
2.7.4

